e2e/pnpm_lockfiles/projects/peer-types/package.json

```diff
@@ -2,11 +2,18 @@
     "name": "test-peer-types",
     "private": true,
     "description": "A package with peer dependencies including: workspace, link, also-in-dev, odd version specifiers",
-    "peerDependencies": {
+    "devDependencies": {
         "@scoped/a": "workspace:*",
         "@scoped/b": "link:../b",
```

Comment:
All of these odd version specified (except `workspace:*`) are no longer valid `peerDependencies` in v10 so I'm changing the test